src/problem/generator/fractionalEquation.ts

```diff
@@ -0,0 +1,306 @@
+import { ProblemGenerator } from "../ProblemGeneratorSpi";
+import { randomInt } from "../util/randomizer";
+import { calculategcd } from "../util/commonDivisor";
+import { SortByAlpha } from "@mui/icons-material";
```

Comment:
this import seems to have slipped in... it can be removed

Reply:
Only Sort by Alpha can be removed.

Reply:
Yes